provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -24,13 +24,38 @@ TV_DOT_GO=${TV_DOT_GO?must be set in the environment.}
 
 printf "package main\n\n" > ${TV_DOT_GO}
 
-for t in ${COMID_TEMPLATES}
-do
-	cocli comid create -t ${t}.json
-	cocli corim create -m ${t}.cbor -t ${CORIM_TEMPLATE} -o corim${t}.cbor
+generate_cbor () {
+	echo "generating cbor using $1 $2"
+	echo $1
+	echo $2
+	cocli comid create -t ${1}.json
+	cocli corim create -m ${1}.cbor -t $2 -o corim${1}.cbor
 	echo "// automatically generated from $t.json" >> ${TV_DOT_GO}
-	echo "var unsignedCorim${t} = "'`' >> ${TV_DOT_GO}
-	cat corim${t}.cbor | xxd -p >> ${TV_DOT_GO}
+	echo "var ${3}${1} = "'`' >> ${TV_DOT_GO}
+	cat corim${1}.cbor | xxd -p >> ${TV_DOT_GO}
 	echo '`' >> ${TV_DOT_GO}
 	gofmt -w ${TV_DOT_GO}
```

Comment:
formatting needs not to be done after each variable is appended, it can be done once at the end

Reply:
It was there initially and i coded without changing the logic, but I agree and hence moved it to end!